app/services/tenor.js

```diff
@@ -17,14 +17,18 @@ export default class TenorService extends Service {
     @tracked columns = null;
     @tracked error = null;
     @tracked gifs = new TrackedArray([]);
-    @tracked searchTerm = '';
-    @tracked loadedType = 'trending';
+    @tracked searchTerm = 'excited';
+    @tracked loadedType = 'search';
```

Comment:
❔ Why the change from an empty search term to 'excited'? This is not required for the migration. 

❔Also, why change from "trending" to search? The new API also supports returning trending GIFs?

Reply:
The API has changed not only in the URL but also in functionality. We can no longer search for "trending" gifs, as the result for that will be gifs representing "Trending stuff", such as statistics graphs or arrows pointing up.

I'm using excited as it is part of the new documentation they provide to show "happy" or "exciting" gifs on the search when there hasn't been any input yet.